extensions/vscode/src/views/homeView.ts

```diff
@@ -979,6 +984,53 @@ export class HomeViewProvider implements WebviewViewProvider, Disposable {
     return commands.executeCommand(Commands.Logs.Focus);
   }
 
+  private async showAssociateGUID() {
+    let urlOrGuid = "";
+    const result = await window.showInputBox({
+      title: "Enter the URL of the Existing Content Item on the Server.",
+      prompt:
+        "You can supply either the content's dashboard URL or the solo URL.",
+      value: urlOrGuid,
+      placeHolder:
+        "For example: https://connect.company.co/connect/#/apps/adffa505-08c7-450f-88d0-f42957f56eff",
```

Comment:
This is already pretty long, I would simplify a bit by just providing the placeholder value.

```suggestion
        "https://connect.company.co/connect/#/apps/adffa505-08c7-450f-88d0-f42957f56eff",
```

Reply:
changed.